src/apps/lwaftr/fragmentv4_hardened.lua

```diff
@@ -267,20 +267,23 @@ function initialize_frag_table(max_fragmented_packets, max_pkt_frag)
       value_type = ffi.typeof(ipv4_reassembly_buffer_t),
       hash_fn = hash_ipv4,
       initial_size = math.ceil(max_fragmented_packets / max_occupy),
-      max_occupancy_rate = max_occupy
+      max_occupancy_rate = max_occupy,
+      memuse_counter = memuse_counter
    }
    return ctable.new(params)
 end
 
 function cache_fragment(frags_table, fragment)
    local key = get_key(fragment)
    local ptr = frags_table:lookup_ptr(key)
+   local ej = false
    if not ptr then
       local reassembly_buf = packet_to_reassembly_buffer(fragment)
-      frags_table:add_with_random_ejection(key, reassembly_buf, false)
+      _, ej = frags_table:add_with_random_ejection(key, reassembly_buf, false)
```

Comment:
I could, but that strikes me as a lot less readable. Call me an Erlanger if you must. :-)